File: src/main/java/br/gov/pf/model/entity/City.java
package br.gov.pf.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

@Entity
@Table(name = "cities")
public class City extends AbstractEntity {

    @Column(name = "description")
    private String description;

    @ManyToOne
    private State state;

    public String getDescription() {
        return this.description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public State getState() {
        return this.state;
    }

    public void setState(State state) {
        this.state = state;
    }
}

File: src/main/java/br/gov/pf/model/entity/City.kt
package br.gov.pf.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

@Entity
@Table(name = "cities")
class City : AbstractEntity() {

    @Column(name = "description")
    var description: String = ""

    @ManyToOne
    var state: State = State()
}